packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -157,16 +157,21 @@ export class QueryExportState {
   decorator?: ((query: Query) => void) | undefined;
   queryBuilderState: QueryBuilderState;
   persistQueryState = ActionState.create();
+  hideExportState?: boolean | undefined;
+  renameExport?: boolean | undefined;
```

Comment:
Are these ever going to be undefined. I see that you initialize them with false in constructor.